# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/WebCodecsUtils.h
# Commit: 45b561797261
# Full Hash: 45b561797261b18d83e66b09c3bf9be9820e4d09
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-03-11 21:13:39
# Description:
#   Bug 1878300 - Only accept vp8 for vp8 codec string. r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D200678
# ==============================================================================

diff -r 6901b5e67c87 -r 45b561797261 dom/media/webcodecs/WebCodecsUtils.h
--- a/dom/media/webcodecs/WebCodecsUtils.h	Mon Mar 11 14:01:17 2024 +0000
+++ b/dom/media/webcodecs/WebCodecsUtils.h	Mon Mar 11 14:16:41 2024 +0000
@@ -236,6 +236,8 @@
 
 nsString ConfigToString(const VideoDecoderConfig& aConfig);
 
+bool IsSupportedVideoCodec(const nsAString& aCodec);
+
 }  // namespace dom
 
 }  // namespace mozilla